src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java

```diff
@@ -513,14 +513,14 @@ public void testPrivateConstructorWithPropertyResolverAndOmitIgnoreModules() thr
     @Test
     public void testNonExistingPropertyName() throws Exception {
         try {
-            loadConfiguration("InputConfigurationLoaderNonexistingProperty.xml");
+            loadConfiguration("InputConfigurationLoaderNonexistentProperty.xml");
             fail("exception in expected");
         }
         catch (CheckstyleException ex) {
             assertEquals("Invalid exception message",
                 "unable to parse configuration stream", ex.getMessage());
             assertEquals("Invalid exception cause message",
-                "Property ${nonexisting} has not been set",
+                "Property ${nonexistent} has not been set",
```

Comment:
`nonexisting` should be a real word. I am fine with `nonexistent` but you didn't change the method name which has the same word in it.
Please verify there are no other instances of this that we can change in this scope.

Reply:
@rnveach done.